Jenkinsfile

```diff
@@ -167,6 +167,8 @@ def initializeEnvironment() {
     sudo apt-get install socat
     pip install --upgrade pip
     pip install -U setuptools
+    sed -i -e 's/pyyaml.*/pyyaml/' ${HOME}/ccm/requirements.txt
+    sed -i -e "s/pyYaml < 6.0/pyYaml/" ${HOME}/ccm/setup.py
```

Comment:
These kinds of inline mods are always somewhat brittle, but fortunately we shouldn't need to do this here.  Just do an install for "pyyaml > 6.0" _after_ the "pip install ${HOME}/ccm" below:

```python
pip install "pyyaml > 6.0"
```

This will do something like the following:

```python
$ pip list
Package    Version
---------- -------
pip        22.0.4
PyYAML     5.4.1
setuptools 56.0.0
$ pip install "pyyaml > 6.0"
Collecting pyyaml>6.0
  Using cached PyYAML-6.0.1-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (723 kB)
Installing collected packages: pyyaml
  Attempting uninstall: pyyaml
    Found existing installation: PyYAML 5.4.1
    Uninstalling PyYAML-5.4.1:
      Successfully uninstalled PyYAML-5.4.1
Successfully installed pyyaml-6.0.1
$ pip list
Package    Version
---------- -------
pip        22.0.4
PyYAML     6.0.1
setuptools 56.0.0
```

The sample above uses a cached copy of PyYAML 6.0.1 (I'd already installed it previously in that venv) but if that's not there it'll download it normally.